mautic/mautic
# Fix campaign form conditions selectbox value

fixes https://github.com/mautic/mautic/issues/9370
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.2
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | Fixes https://github.com/mautic/mautic/issues/9370




#### Description:

When you use the form field condition,  if you filter on a select field. The filter is on the value of the field and not on Label.  
![image](https://user-images.githubusercontent.com/49391402/97539844-f7ff8500-19c2-11eb-9f48-e17f8dd41a2f.png)



#### Steps to test this PR:
Step to reproduce : 
1. Create from with select form field with value different than label
2. Create a campaign form with form value condition
3. Select from and your created form field to condition
4. Save it and re-open condition
5. You should see values instead labels in selectbox without this fix
